templates/components/api-server/template/server/middleware.json

```diff
@@ -10,6 +10,26 @@
         "credentials": true,
         "maxAge": 86400
       }
+    },
+    "helmet#xssFilter": {
+      "params": {
+        "setOnOldIE": true
+      }
+    },
+    "helmet#frameguard": {
+      "params": "deny"
```

Comment:
I'd prefer to wrap the value in an array, to make it more clear how to pass more than one item.

```js
// default in the scaffolding
"params": ["deny"]
// an example of a custom config
"params": ["allow-from", "http://example.com"]
```